src/docs/timestamp-global.dox

```diff
@@ -42,35 +38,37 @@ WT_CONNECTION::set_timestamp method, including constraints.
 
 | Timestamp | Constraints | Description |
 |-----------|-------------|-------------|
-| durable_timestamp | <= oldest | Reset the maximum durable timestamp (see @ref timestamp_prepare for discussion of the durable timestamp). |
+| durable_timestamp | none | temporarily set the system's maximum durable timestamp, bounding the timestamp returned by \c all_durable. |
 | oldest_timestamp | <= stable; may not move backward, set to the value as of the last checkpoint during recovery | Inform the system future reads and writes will never be earlier than the specified timestamp. |
-| stable_timestamp | may not move backward, set to the recovery timestamp during recovery | Inform the system checkpoints should not include commits newer than the specified timestamp. |
+| stable_timestamp | may not move backward, set to the recovery timestamp during recovery and the specified timestamp during rollback-to-stable | Inform the system checkpoints should not include commits newer than the specified timestamp. |
```

Comment:
RTS uses the already set stable timestamp; you can't, currently at least, set a different timestamp. So it doesn't change stable. Consequently I think this extra text is confusing and not really necessary.

Reply:
Changed to "set to the stable timestamp during rollback-to-stable and recovery" in 8bac211.